design-proposals/orch-cli.md

```diff
@@ -0,0 +1,229 @@
+# Design Proposal: Orchestrator Command Line Interface
+
+Author(s): Scott Baker
+
+Last updated: 2025-04-29
+
+## Abstract
+
+This ADR describes the design of a command line interface (CLI) for the
+Orchestrator. The CLI is a cross-component tool that may be used from a Linux, Windows, or
+Mac environment, such as from the administrator's laptop. The CLI may also may be leveraged
+as a building block in other environments such as CI/CD or testing.
+
+The CLI described here uses the `Catalog CLI` as a strating point and inspiration. However, this
+ADR assumes that syntax and semantics of the CLI may diverge from the existing `Catalog CLI` and
+this ADR is not constrained by that existing implementation. As such, examples given in this
+ADR may not necessarily work with the existing catalog CLI.
+
+## Proposal
+
+The CLI implements a set of verbs, a set of nouns, and a set of common options that may
+be used to interact with the orchestrator.
```

Comment:
I think we will need to outline what are the differences between this new cli tool and existing mage targets i.e., if new functionality is added is implemented as a mage target or in the cli

Reply:
Adding a paragraph to try to disambiguate the purpose.